### PR TITLE
Updated Top-Level Requirements.md

### PR DESCRIPTION
**Pull request by AdamuKaapan**
Added the extra requirements that I proposed in the initial pull request. These requirements are regarding moving the arm out of the way of the bin during the dumping procedure.

### CODE REVIEW

File: Top-Level Requirements.md [hunk 1]
```diff
@@ -77,10 +77,12 @@
 * 8.8 Upon completing the mining procedure, the mining subsystem **shall**:
     * 8.8.1 Define an “obstacle” representing the digging hole and send it to the navigation subsystem.
     * 8.8.2 Return the mining arm to the starting position.
+* 8.9 The mining subsystem **shall** provide an ability to move the digging arm to a predefined dumping position to enable bin movement without interference
```
Comment (collinsc) [inline Top-Level Requirements.md]: this is a little messy, can we strike 8.9 and reword 8.8.2 to cover it?

The goal we are trying to accomplish is to clean up after we finish, and that's the job of requirement 8.8.

Would this work:
     * 8.8.2 Return the mining arm to an ending position clear of the range of motion of the dumping bin.

File: Top-Level Requirements.md [hunk 2]
```diff
@@ -77,10 +77,12 @@
...
 ### 9.  Dumping Subsystem
 * 9.1 As a precondition the rover **shall** be facing east, relatively close to the dumping receptacle.
 * 9.2 The dumping subsystem **shall** perform the localization procedure prior to the start of the  dumping procedure.
 * 9.3 The dumping procedure **shall**:
     * 9.3.1 Orient itself and turn to face east.
     * 9.3.2 Back up to within a set tolerance of the wall, maintaining eastern orientation.
-    * 9.3.3 Raise the mining bin and dump the regolith at a set speed.
+    * 9.3.3 Move the digging arm out of the bin to the predefined dumping position.
```
Comment (collinsc) [inline Top-Level Requirements.md]: I really don't think this is dumping's job. Digging should clean up after itself when it's done.

We had a conversation about this in a meeting, and we decided it is the job of the individual subsystems to handle their own cleanup, it adheres more to RAII.

I think we should revert.

File: Top-Level Requirements.md [hunk 3]
```diff
@@ -77,10 +77,12 @@
 * 8.8 Upon completing the mining procedure, the mining subsystem **shall**:
```
Comment (collinsc) [inline Top-Level Requirements.md]: I think there is a slight omission here. Mining has the chance to be preempted. I think 8.8 should be reworded to:

* 8.8 Upon completing the mining procedure or user preemption, the mining subsystem **shall**:

**Review by collinsc — CHANGES_REQUESTED** [review]

File: Top-Level Requirements.md (hunk 3 above)
Comment (AdamuKaapan) [reply Top-Level Requirements.md]: Agreed, yeah, I'll change that.

File: Top-Level Requirements.md (hunk 1 above)
Comment (AdamuKaapan) [reply Top-Level Requirements.md]: Yeah, sounds good.

File: Top-Level Requirements.md (hunk 2 above)
Comment (AdamuKaapan) [reply Top-Level Requirements.md]: Yup, as per conversation in slack I'll go ahead and strike this.

**Review by RyanBerge — APPROVED** [review]

**Review by collinsc — APPROVED** [review]
Good! thanks for changing that.

**Comment by collinsc** [comment]
@AdamuKaapan @RyanBerge I don't think auto merging is turned on so one of you might have to write the changes. It's not in my permission set.

**Pull request merged**